pkg/cmd/config/command.go

```diff
@@ -75,21 +87,6 @@ func NewCommand(ctx context.Context) *cli.Command {
 					return nil
 				},
 			},
-			{
```

Comment:
This command was initially to prove out that we can list the services installed on a Cluster. Now that this functionality has moved into the `generate` command, I opted to remove it.

Reply:
I wonder if we still want this as a `hidden` command?